### PR TITLE
Proximity migration

### PR DESCRIPTION
**Pull request by Tuxosaurus**
Screen Shot 2019-07-29 at 16 41 38

### CODE REVIEW

File: src/proximity/index.tsx [hunk 1]
```diff
@@ -1,63 +1,21 @@
-import React from 'react'
```
Comment (pith) [inline src/proximity/index.tsx]: You can rename this file `Proximity.unit.tsx`

File: package.json [hunk 2]
```diff
@@ -87,6 +87,7 @@
     "country-telephone-data": "0.6.0",
     "exenv": "1.2.2",
     "focus-trap": "5.0.0",
+    "jest-styled-components": "6.3.3",
```
Comment (pith) [inline package.json]: You can rebase on the `styled-components` branche. I already imported it and setup the config

File: src/proximity/index.tsx (hunk 1 above)
Comment (Tuxosaurus) [reply src/proximity/index.tsx]: hum, this is the new index file. The unit test file has already been renamed to `Proximity.unit.tsx`

Comment (Tuxosaurus) [reply src/proximity/index.tsx]: @pith is it OK?

Comment (pith) [reply src/proximity/index.tsx]: Yes, sorry stupid comment.

**Pull request merged**